app/templates/@layout.latte

```diff
@@ -7,6 +7,16 @@
     <meta name="author" content="">
     <link rel="icon" href="/favicon.ico">
 
+    <script n:syntax="off">
```

Comment:
nemá tohle být v externím souboru .js?

Reply:
No, teoreticky ano - ideálně by to bylo v `DarkModeToggle.ts`. Akorát externí `.js` soubory se načítají až později při parsování stránky, což způsobí, že ta stránka v dark mode krátce problikne bíle než se to externí js načte a spustí. A to je extrémně nepříjemný (hlavně při navigaci, protože kvůli latte prakticky jakákoliv interakce znamená reload), takže je potřeba tenhle kus mít natvrdo tady, aby se to rovnou načetlo v tmavé variantě...